images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -17,7 +19,7 @@ $msys2Uri = ((Invoke-RestMethod $msys2_release).assets | Where-Object {
 
 $msys2File = "$env:TEMP\msys2.tar.xz"
 
-# Download the latest msys2 x86_64
+# Download the latest msys2 x86_64, filename includes release date
 Write-Host "Starting msys2 download using $($msys2Uri.split('/')[-1])"
 (New-Object System.Net.WebClient).DownloadFile($msys2Uri, $msys2File)
```

Comment:
I suggest to use helper `Start-DownloadWithRetry` (https://github.com/actions/virtual-environments/blob/master/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L218) here

Reply:
@maxim-lobanov  Good idea.  Done.